src/main/java/com/example/restapi/controller/EmployeeApiController.java

```diff
@@ -0,0 +1,60 @@
+package com.example.restapi.controller;
+
+import com.example.restapi.entities.Employee;
+import com.example.restapi.repository.EmployeeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api/v1")
+public class EmployeeApiController {
+	@Autowired
+	private EmployeeRepository employeeRepository;
+
+	@GetMapping("/employee")
+	public List<Employee> getEmployeeList() {
+		return this.employeeRepository.findAll();
+	}
+
+	@GetMapping("/employee/{employeeId}")
```

Comment:
/employee*s*/{employeeId}
refer to this Resource Naming https://restfulapi.net/resource-naming/

Reply:
Fixed